### PR TITLE
Add FR README

### PR DESCRIPTION
**Pull request by Biokax**
Hi,

this is a proposal for the french translation of the readme file.
I will try to make natives people reviews this before merge

may check links because the copy of the readme was old

### CODE REVIEW

File: README_FR.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+- [Tablettes supportées](https://opentabletdriver.net/Tablets)
+
+# Installation
+
+- [Windows](https://opentabletdriver.net/Wiki/Install/Windows)
+- [Linux](https://opentabletdriver.net/Wiki/Install/Linux)
+- [MacOS](https://opentabletdriver.net/Wiki/Install/MacOS)
+
+# Exécuter un binaire OpenTabletDriver
```
Comment (Mrcubix) [inline README_FR.md:27]: I don't think you can translate it that way, at the end of the day, some french users are going to ask themselves, what is software binary? The answer is simple, in this case, it's an executable.
So i a think a better translation would be simply the following:
```suggestion
# Executer OpenTabletDriver
```

File: README_FR.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  - Support de script (sortie json)
+- Positionnement absolu du curseur
+  - Zone de l'écran ainsi que la zone de la tablette
+  - Décalages par rapport au centre
+  - Rotations précises de la zone
+- Positionnement relatif du curseur
+  - Sensibilité horizontal et vertical (px/mm)
+- Raccourcis du stylet
+  - Raccourcis de pression de la pointe
+  - Raccourcis express
```
Comment (Mrcubix) [inline README_FR.md:102]:
```suggestion
  - Raccourcis touches express
```

File: README_FR.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+```
+sudo ln -s /lib64/libevdev.so.2 /lib64/libevdev.so
+```
+- GTK+3
+
+Pour pouvoir build sur Linux, exécutez le 'build.sh' fourni. Ceci va également exécuter
+la command 'dotnet publish' utilisée pour build le paquet AUR, 
+cela va également produire des binaires utilisables dans 'OpenTabletDriver/bin'.
+
+Pour build un ARM Linux, exécutez le 'build.sh' fourni
```
Comment (Mrcubix) [inline README_FR.md:67]:
```suggestion
Pour build sur ARM Linux, exécutez le 'build.sh' fourni 
```

**Review by Mrcubix — CHANGES_REQUESTED** [review]
Most of the translation is fine, i would however ask you to make the following changes

File: README_FR.md (hunk 1 above)
Comment (Biokax) [reply README_FR.md:27]: Indeed that better

File: README_FR.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+- [Tablettes supportées](https://opentabletdriver.net/Tablets)
+
+# Installation
+
+- [Windows](https://opentabletdriver.net/Wiki/Install/Windows)
+- [Linux](https://opentabletdriver.net/Wiki/Install/Linux)
+- [MacOS](https://opentabletdriver.net/Wiki/Install/MacOS)
+
+# Executer OpenTabletDriver
```
Comment (Mrcubix) [inline README_FR.md:27]: small mistake on my part missing an accent is missing
```suggestion
# Exécuter OpenTabletDriver
```

File: README_FR.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+Paquets requis (certains paquets peuvent-être pré-installés pour votre distribution)
+
+- libx11
+- libxrandr
+- libevdev2
+
+Sur Fedora ou d'autres distributions, libevdev2 peut-être présentes,
+mais requière un symlink car le nom du binaire est différent de ce que .NET attend.
+
+Commande symlink utilisée sur Fedora 34:
```
Comment (Sublimelime) [inline README_FR.md:56]: This part about symlinking the library potentially shouldn't be included as it was fixed in #1505 and shouldn't be necessary anymore.... though we kinda have the issue that it's fixed for 0.6 but is still necessary on stable 0.5. @X9VoiD or @InfinityGhost should advise whether or not to include it.

Comment (InfinityGhost) [reply README_FR.md:56]: It is no longer required so I wouldn't mention it anymore.

Comment (Biokax) [reply README_FR.md]: I removed it

**Review by Sublimelime — APPROVED** [review]
Note for the future that it's unnecessary to merge our master into your branch/master unless there's merge conflicts you have to resolve, git handles it fine.

**Review by InfinityGhost — APPROVED** [review]

File: README_FR.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
+[![Actions Status](https://github.com/OpenTabletDriver/OpenTabletDriver/workflows/.NET%20Core/badge.svg)](https://github.com/OpenTabletDriver/OpenTabletDriver/actions) [![CodeFactor](https://www.codefactor.io/repository/github/OpenTabletDriver/OpenTabletDriver/badge/master)](https://www.codefactor.io/repository/github/OpenTabletDriver/OpenTabletDriver/overview/master) [![Total Download Count](https://img.shields.io/github/downloads/OpenTabletDriver/OpenTabletDriver/total.svg)](https://github.com/OpenTabletDriver/OpenTabletDriver/releases/latest)
+
+# OpenTabletDriver
+
+[English](README.md) | [한국어](README_KO.md) | [Español](README_ES.md) | [Русский](README_RU.md) | [简体中文](README_CN.md) | Français
+
+OpenTabletDriver est un driver de tablette en mode utilisateur, open source et multiplateforme. Le but d'OpenTabletDriver est d'être compatible avec le plus de plateforme possibles, et ce avec une interface graphique utilisateur facilement configurable.
```
Comment (Mrcubix) [inline README_FR.md:7]:
```suggestion
OpenTabletDriver est un driver de tablette en mode utilisateur, open source et multiplateforme. Le but d'OpenTabletDriver est d'être compatible avec le plus de plateforme possibles, et ce grâce à une interface graphique utilisateur facilement configurable.
```

File: README_FR.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+Paquets requis (certains paquets peuvent-être pré-installés pour votre distribution)
+
+- libx11
+- libxrandr
+- libevdev2
+- GTK+3
+
+Pour pouvoir build sur Linux, exécutez le 'build.sh' fourni. Ceci va également exécuter
+la command 'dotnet publish' utilisée pour build le paquet AUR, 
+cela va également produire des binaires utilisables dans 'OpenTabletDriver/bin'.
```
Comment (Mrcubix) [inline README_FR.md:56]:
```suggestion
cela va également produire des exécutables utilisables dans 'OpenTabletDriver/bin'.
```

**Review by Mrcubix — CHANGES_REQUESTED** [review]
Second review of stuff i've missed, Should be merged after the suggested modifications are applied.

**Pull request merged**